python/cuda_parallel/README.md

```diff
@@ -6,7 +6,16 @@ Please visit the documentation here: https://nvidia.github.io/cccl/python.html.
 
 ## Local development
 
+First-time installation:
+
+```bash
+pip3 install ./cuda_cccl
+pip3 install ./cuda_parallel[test]
```

Comment:
Can this be editable if necessary? Wouldn't a regular install here and then an editable install below would lead to two copies of the package in the environment?

Reply:
It can! Thanks for asking. Previously I was incorrectly thinking that one pass without `-e` is required. I tried again fresh and it turns out it actually does work on first pass.

I restored the original README.md: commit 9ed6036bde6a1e08d44835b88a8194825e48cc3c

> Wouldn't a regular install here and then an editable install below would lead to two copies of the package in the environment?

From what I can tell, the 2nd install clobbers the previous one.